.github/workflows/pr-build.yml

```diff
@@ -47,8 +47,8 @@ jobs:
     container: 
       image: hypertrace/helm-gcs-packager:0.3.1
       credentials:
-        username: ${{ secrets.DOCKERHUB_USERNAME }}
```

Comment:
Whoops, didn't realize these were still floating around. I'm going to remove those secrets entirely (no longer valid anyway)